src/coreclr/tools/aot/crossgen2/crossgen2.targets

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!-- 
+    Works around the fact that the Content items defined in crossgen2.props get cleaned 
+    after subsequent build runs. Without this target, the build will fail when we 
+    try to use the inbuild crossgen as it will try to look for native libs that aren't there.
+
+    Unsure if this is a bug in the SDK or if we are doing something wrong. Warrants more investigation.
```

Comment:
Can we open up a follow-up issue and link it here if we're going to investigate further?

Reply:
I found the reason, we need to set `ExcludeFromSingleFile=true` or the SDK won't deploy these files next to the bundle. This is sort of documented in https://github.com/dotnet/designs/blob/main/accepted/2020/single-file/design_3_0.md and https://github.com/dotnet/sdk/issues/3765#issuecomment-544685460